### PR TITLE
inspiron 14 5420: initial

### PR DESCRIPTION
**Pull request by bagelwaffle**
###### Things done

- [x] Tested the changes in your own NixOS Configuration
- [x] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

**Comment by bagelwaffle** [comment]
oh! so apparently tlp may conflict with the built GNOME power management thing, so should I remove the tlp section of the file or is there something I should add to it

**Comment by bagelwaffle** [comment]
actually, I think I'll just remove TLP and say in the README that it may conflict with GNOME power management

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ See code for all available configurations.
 | [BeagleBoard PocketBeagle](beagleboard/pocketbeagle)                | `<nixos-hardware/beagleboard/pocketbeagle>`        |
 | [Deciso DEC series](deciso/dec)                                     | `<nixos-hardware/deciso/dec>`                      |
 | [Dell G3 3779](dell/g3/3779)                                        | `<nixos-hardware/dell/g3/3779>`                    |
+| [Dell Inspiron 14 5420](dell/inspiron/14-5420)                      | `<nixos-hardawre/dell/inspiron/14-5420>`           |
```
Comment (Mic92) [inline README.md:93]: Please also add a line in flake.nix

Comment (bagelwaffle) [reply README.md:93]: It has been done!

File: dell/inspiron/14-5420/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- CPU: Intel® Core™ i7-1260P Processor
+- GPU: Intel® Iris® Xe Graphics
+
+### Other Info
+`fwupd` is used to receive firmware updates from vendors. This device is in its [supported devices list](https://fwupd.org/lvfs/devices/)
+
+`thermald` is used to achieve better thermal behaviour
+
+##### TLP
+TLP is a battery power optimizer for laptops. You may enable it with `services.tlp.enable = true;`, however, please note that it may conflict with GNOME's power management. Enabling TLP may require additional configurations.
```
Comment (Mic92) [inline dell/inspiron/14-5420/README.md:13]: We do this already in ../../../common/pc/laptop if the gnome power manager is not present.

Comment (bagelwaffle) [reply dell/inspiron/14-5420/README.md]: That is now fixed as well

**Pull request merged**